internal/admin/repository/eventsourcing/eventstore/iam.go

```diff
@@ -414,6 +419,9 @@ func (repo *IAMRepository) GetDefaultLoginTexts(ctx context.Context, lang string
 	contents, ok := repo.TranslationFileContents[lang]
 	if !ok {
 		contents, err := repo.readTranslationFile(fmt.Sprintf("/i18n/%s.yaml", lang))
+		if os.IsNotExist(err) {
+			contents, err = repo.readTranslationFile(fmt.Sprintf("/i18n/%s.yaml", defaultLang))
+		}
```

Comment:
please use default language from system defaults

Reply:
and shouldn't there be a change for mgmt / org repo as well?

Reply:
done